lib/studies/reply/inbox.dart

```diff
@@ -52,26 +55,38 @@ class _AdaptiveNavState extends State<AdaptiveNav> {
       'Freelance': _folderIconAssetLocation,
     };
 
-    return ResponsiveWidget(
-      desktopScreen: _BuildDesktopNav(
-        selectedIndex: _selectedIndex,
-        extended: true,
-        destinations: _navigationItems,
-        onItemTapped: _onDestinationSelected,
-      ),
-      tabletScreen: _BuildDesktopNav(
-        selectedIndex: _selectedIndex,
-        extended: false,
-        destinations: _navigationItems,
-        onItemTapped: _onDestinationSelected,
-      ),
-      mobileScreen: _BuildMobileNav(
-        selectedIndex: _selectedIndex,
-        destinations: _navigationItems,
-        folders: _folders,
-        onItemTapped: _onDestinationSelected,
-      ),
-    );
+    switch (isTablet) {
```

Comment:
I think using a switch for a bool is a bit odd. I'd just use if statements. And you could just return directly in the if. 

```
if (isTablet) {
  return ...
}else if (isDesktop) {
  return ...
} else {
  return ...
}
```

Reply:
That makes sense thanks!